db_setup/mkdb.sql

```diff
@@ -69,30 +68,27 @@ CREATE TABLE BoatConfiguration (
   OprettetDato datetime,
   RedigeretDato datetime,
   Kommentar varchar(1000),
-  Initialer varchar(10)
 );
 
 DROP TABLE IF EXISTS BoatRights;
 CREATE TABLE BoatRights (
   boat_type int(11) NOT NULL,
   required_right varchar(30) NOT NULL,
   requirement varchar(10),
+  FOREIGN KEY (required_right) REFERENCES MemberRightType(member_right) ON DELETE CASCADE ON UPDATE CASCADE,
   PRIMARY KEY (boat_type,required_right)
 );
 
 DROP TABLE IF EXISTS BoatType;
 CREATE TABLE BoatType (
-  id int(11) NOT NULL AUTO_INCREMENT,
   Name varchar(100),
   Seatcount int(11),
   Description varchar(1000),
   Category int(11),
   Created datetime,
   Updated datetime,
-  Initials varchar(10),
   rights_subtype CHAR(20),
-  PRIMARY KEY (id),
-  KEY gruppenavn (`Name`)
+  PRIMARY KEY (`Name`)
```

Comment:
... men betydningsbærende nøgler er en ualmindelig skidt idé.

Læs f.x. https://www.techrepublic.com/blog/10-things/10-tips-for-choosing-between-a-surrogate-and-natural-primary-key/

I det her tilfælde er det ekstra dumt: Vi genbruger bådnavne. Så man vil ikke kunne se forskel på, om en tur er roet i den gamle eller den nye båd med samme navn.